src/actions/Account.actions.js

```diff
@@ -76,6 +81,9 @@ export const loadAccountDetails =
           ohmStake: +stakeAllowance,
           ohmUnstake: +unstakeAllowance,
         },
+        migrate: {
+          unstakeAllowance: unstakeAllowanceSohm,
+        },
```

Comment:
Can you put stakeAllowance in here as well?
`getApproval` in `Migrate.actions.js` populates the state with that as well. Might as well haaave everything from the beginning.